### PR TITLE
Miscellany

### LINKED ISSUE #112: Embolden control labels.
# Test notes
* Control labels look elegant af.

### LINKED ISSUE #113: Update UI to allow easy de-selection of bars.
# Test notes
* Clicking in the white space of the chart canvas after clicking one or more bars deselects all bars and resets the listing.
* Clicking the _Reset listing_ button does the same.

### LINKED ISSUE #116: Sync query age with relevant query statuses.
i.e. define a subset of query statuses that will receive a query age, e.g. `['Open']`

# Test notes
* Verify that the query statuses specified here are presented by query age category rather than by query status when _Status Group_ is set to _Query Age_.

### LINKED ISSUE #117: Replace query age categories with query age cutoffs.
like `[14, 28, 56, 112]` instead of `['0-2 weeks', '2-4 weeks', ...]`

# Test notes
* Add and remove query age cutoffs and verify the _Query Age_ filter and _Status Group_ update as expected.

### LINKED ISSUE #118: Use query status colors when stratifying by query age.
for query statuses that are not relevant to query age (i know that's worded poorly)

# Test notes
* When _Status Group_ is set to _Query Age_ the colors of _Answered_, _Closed_, and _Cancelled_ categories should match the colors of the same categories when _Status Group_ is set to _Query Status_.

### LINKED ISSUE #119: Replace query recency categories with query recency cutoffs.
`[7, 14, 30]` instead of `['0-7 days', '7-14 days', '14-30 days']`

# Test notes
* Set _recency_category_col_ in CAT to null.
* Render chart.
* The _Query Recency_ filter should still appear with the categories `[0-7 days', '7-14 days', '14-30 days', 'N/A']`.
* Add and remove cutoffs and verify the filter updates as expected.

### LINKED ISSUE #120: Add additional chart options to renderer settings.
- `color_by_col`: `querystatus`
- `bar_arrangement`: `grouped`

# Test notes
* The settings form in CAT should allow the specification of the initial _Status Group_ variable and the initial bar arrangement.

### PR DESCRIPTION
**Pull request by samussiah**
# Issues
Closes #112 
Closes #113 
Closes #116 
Closes #117 
Closes #118 
Closes #119 
Closes #120

### CODE REVIEW

File: src/chart/onInit/defineNewVariables.js [hunk 1]
```diff
@@ -1,51 +1,42 @@
...
-                    d['Query Age Category'] = d[this.config.status_col];
-                    break;
-            }
+        //Define query age.
+        if (this.config.age_statuses.indexOf(d[this.config.status_col]) < 0)
+            d[queryAgeCol] = d[this.config.status_col];
+        else {
+            const age = +d[this.config.age_col];
+            this.config.ageRanges.forEach((ageRange, i) => {
+                if (i === 0 && ageRange[0] <= age && age <= ageRange[1])
```
Comment (pburnsdata) [inline src/chart/onInit/defineNewVariables.js:18]: beautiful

File: src/chart/onLayout/addControlTooltips.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
+export default function addControlTooltips() {
+    const tooltips = {
+        //other controls
+        'Status Group':
```
Comment (pburnsdata) [inline src/chart/onLayout/addControlTooltips.js:4]: I might like these hovers better than the info bubbles in the heatmap - don't take up space but are still very noticeable

Comment (pburnsdata) [reply src/chart/onLayout/addControlTooltips.js:4]: Some nice writing here - powerful, concise, clear - moving. Shed a few tears during this review

File: src/chart/onResize/addBarClick/mouseoverAttrib.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+export default function mouseoverAttrib(element) {
+    const selection = d3.select(element);
+    selection.attr({
+        width: function(d) {
+            return this.getBBox().width - 2.5;
```
Comment (pburnsdata) [inline src/chart/onResize/addBarClick/mouseoverAttrib.js]: Yep. Much better

**Review by pburnsdata — APPROVED** [review]

**Comment by pburnsdata** [comment]
Great work on this - much cleaner in code and appearance

**Comment by danedexF5** [comment]
Passed feature testing

**Pull request merged**